src/lib/getCoverage.ts

```diff
@@ -36,15 +40,19 @@ const getCoverage = async (options?: Options): Promise<CoverageData> => {
     ignoreUnread: ignoreUnreadAnys
   } = options || {};
 
-  const { anys, fileCounts, totalCount, correctCount } = await lint(".", {
-    strict,
-    debug,
-    ignoreFiles,
-    ignoreCatch,
-    enableCache,
-    ignoreUnreadAnys,
-    fileCounts: true
-  });
+  const projectPath = tsProjectFile || ".";
```

Comment:
`tsProjectFile` should already come as `"."` as you gave the default, so, in theory, you shouldn't need this.

Reply:
I've tried the below:

```ts
const { anys, fileCounts, totalCount, correctCount } = await lint(
    tsProjectFile,
    {
      strict,
      debug,
      ignoreFiles,
      ignoreCatch,
      enableCache,
      ignoreUnreadAnys,
      fileCounts: true
    }
  );
```

However, the unit tests are then failing:
![image](https://user-images.githubusercontent.com/1135454/115763092-0e97ff00-a39c-11eb-865d-e8a0463fefca.png)

Reply:
Yeah, but you're right: the unit should work without the binary so having a default is definitely better. What you could do then (just to have slightly more elegant code) highlighted in a different comment